exercises/ex1.cpp

```diff
@@ -11,7 +11,7 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
+  /*if (week == 1)
```

Comment:
Potresti cancellare questa sezione di commento in quanto non aggiunge informazioni utili al fine del programma.